src/overlays/actors/ovl_Obj_Tsubo/z_obj_tsubo.h

```diff
@@ -12,7 +12,7 @@ typedef struct ObjTsubo {
     /* 0x0000 */ Actor actor;
     /* 0x014C */ ObjTsuboActionFunc actionFunc;
     /* 0x0150 */ ColliderCylinder collider;
-    /* 0x019C */ s8 objTsuboBankIndex;
+    /* 0x019C */ s32 objTsuboBankIndex;
 } ObjTsubo; // size = 0x01A0
```

Comment:
possible size note change

Reply:
I would just delete size notes, they wont be accurate with 64bit anyway.